src/Explorer/Controls/NotebookViewer/__snapshots__/NotebookMetadataComponent.test.tsx.snap

```diff
@@ -180,14 +180,14 @@ exports[`NotebookMetadataComponent renders un-liked notebook 1`] = `
       Invalid Date
     </Text>
     <Text>
-      <StyledIconBase
+      <Icon
         iconName="RedEye"
       />
        
       0
     </Text>
     <Text>
-      <StyledIconBase
```

Comment:
>StyledIconBase [](start = 7, length = 14)

is this just a rename needed bcoz o the upgrade? or are we changing the control -if so why?

Reply:
This is due to the updagrade of office ui pakage.